src/static/login.html

```diff
@@ -11,7 +11,29 @@
 
         (function(console) {
             var environment = window.environment || { };
+            var oauth = window.oauth || null;
```

Comment:
Okay, good to know ... I thought that with this oauth flow the user only ever saw a redirect and nothing else, so hence my confusion.